### PR TITLE
chore: apply review comments @W-8856585@

### PR DESCRIPTION
**Pull request by peternhale**

### CODE REVIEW

File: src/hubAuth.ts [hunk 1]
```diff
@@ -34,20 +34,16 @@ const DEFAULT_INSTANCE_URL = 'https://login.salesforce.com';
...
-      const newKeyContents = env.getString('TESTKIT_JWT_KEY', '').replace(header, '').replace(footer, '');
-      // check to see if newlines were replaced with spaces
-      keyLines = newKeyContents.trim().split(/\s/);
-      if (keyLines.length <= 1) {
-        // still one big string, split into 64 byte chucks
-        keyLines = [header, ...(newKeyContents.match(/.{1,64}/g) as string[]), footer];
-      } else {
-        keyLines = [header, ...keyLines, footer];
-      }
+      // strip out header, footer and spaces
```
Comment (peternhale) [inline src/hubAuth.ts:42]: simplified

File: src/hubAuth.ts [hunk 2]
```diff
@@ -73,13 +69,21 @@ export const testkitHubAuth = (homeDir: string): void => {
     const jwtKey = path.join(homeDir, 'jwtKey');
     fs.writeFileSync(jwtKey, formatJwtKey());
 
-    shell.exec(
+    const results = shell.exec(
```
Comment (peternhale) [inline src/hubAuth.ts:72]: There is a [bug in shelljs](https://github.com/shelljs/shelljs/issues/981) that ignore `fatal: true` when not set globally, so for now capture results and throw if the command fails.

Comment (mshanemc) [reply src/hubAuth.ts:72]: good catch.

File: src/hubAuth.ts [hunk 3]
```diff
@@ -73,13 +69,21 @@ export const testkitHubAuth = (homeDir: string): void => {
...
       `sfdx auth:jwt:grant -d -u ${env.getString('TESTKIT_HUB_USERNAME', '')} -i ${env.getString(
         'TESTKIT_JWT_CLIENT_ID',
         ''
       )} -f ${jwtKey} -r ${env.getString('TESTKIT_HUB_INSTANCE', DEFAULT_INSTANCE_URL)}`,
-      { silent: true, fatal: true }
+      { silent: true }
     );
+    if (results.code) {
```
Comment (mshanemc) [inline src/hubAuth.ts:79]: preference:`results.code !== 0` is more approachable

**Review by mshanemc — APPROVED** [review]

**Review by mshanemc — APPROVED** [review]

**Pull request merged**